doc/sphinx/generate_doc.py

```diff
@@ -132,8 +132,7 @@ def generate_doc(config_path: str, output_directory: str = "./ref/", package_roo
     namespaces = [n for pack in config.values() for n in pack.get("namespaces", {})]
 
     for namespace in namespaces:
-        if project_pattern and not any(namespace.startswith(p) for p in project_pattern):
-            _LOGGER.info("Skip project %s", namespace)
+        if project_pattern and not namespace.startswith(project_pattern):
```

Comment:
`project_pattern` is a passed as a string like `azure-mgmt-advisor`, so by doing the `p for p in project_pattern` we're iterating over the letters here and comparing them to the namespace.

Also removing the logging here since it's pretty noisy and we can see what docs we're building based on the logger below